### PR TITLE
Add UK L2 hospitalization data

### PR DESCRIPTION
**Pull request by l2edzl3oy**
Replace existing Scotland data source with [UK government data source](https://coronavirus.data.gov.uk/healthcare).

Add England, Northern Ireland, and Wales hospitalization data using the same UK government data source.

Addresses #232 .

### CODE REVIEW

**Comment by owahltinez** [comment]
The failing unit test is unrelated to this PR and has been fixed in main. Can you please rebase this branch to all the other tests can run?

File: src/pipelines/hospitalizations/gb_authority.py [hunk 1]
```diff
@@ -21,44 +21,6 @@
 from uk_covid19 import Cov19API
 
 
-class ScotlandDataSource(DataSource):
```
Comment (themonk911) [inline src/pipelines/hospitalizations/gb_authority.py:24]: It looks like this was getting L3 information, but the replacement is L2.

**Review by themonk911 — CHANGES_REQUESTED** [review]
@l2edzl3oy I think that you are replacing an L3 source with L1/L2, perhaps we can append the new sources instead? Do you know if the API provides hospitalization for L3?

**Comment by owahltinez** [comment]
@l2edzl3oy are you still interested in pursuing this PR? Have you seen the changes requested by @themonk911?

**Comment by l2edzl3oy** [comment]
> @l2edzl3oy are you still interested in pursuing this PR? Have you seen the changes requested by @themonk911?

So sry @owahltinez ! Been caught up with US elections work over the past week. I aim to finish this within the next week. Thanks for the follow up.

File: src/pipelines/hospitalizations/gb_authority.py (hunk 1 above)
Comment (l2edzl3oy) [reply src/pipelines/hospitalizations/gb_authority.py:24]: > @l2edzl3oy I think that you are replacing an L3 source with L1/L2, perhaps we can append the new sources instead? Do you know if the API provides hospitalization for L3?

So sorry @themonk911 ! The data replacement as a result of this code change is intended specifically for the `GB_SCT` key (and only that, not other `GB_SCT_*` keys one level below). I will undo the deletion.

Expected end state is to leave the `GB_SCT_*` (L3) data untouched, but override the `GB_SCT` (L2) data with the data from the UK govt data source. Per the [documentation](https://github.com/GoogleCloudPlatform/covid-19-open-data/blob/main/src/README.md#data-pipeline), this means that I should leave the current Scotland data source in `config.yaml` untouched, and only append the new UK L2 data source **after** the Scotland data source. The expected end state is then achieved. Is my understanding right?

As for your 2nd question - sadly I don't think the API provides hospitalization data for L3.

Comment (themonk911) [reply src/pipelines/hospitalizations/gb_authority.py:24]: ^That sounds good to me - shame the API doesn't do L3. If you are interested in a another PR, you can look at the change in the excel parsing for the L3 data for GB_SCT_* :)

**Comment by l2edzl3oy** [comment]
> The failing unit test is unrelated to this PR and has been fixed in main. Can you please rebase this branch to all the other tests can run?

Sure, have rebased it. Unit tests have passed now :)

File: src/pipelines/hospitalizations/gb_authority.py (hunk 1 above)
Comment (l2edzl3oy) [reply src/pipelines/hospitalizations/gb_authority.py]: Ok I've edited as suggested. Thanks for the heads up - will take a look and see if there are alternative data sources for that too.

**Review by themonk911 — APPROVED** [review]
LGTM, thanks for the PR!

**Comment by owahltinez** [comment]
Great, thanks everyone! Merging this now.

**Pull request merged**